internal/helper/slowmessage/slowmessage.go

```diff
@@ -19,6 +20,7 @@ type CallbackFunc func()
 // Do calls sf. If threshold time has passed, cb is called. Note that this
 // call will be made concurrently to sf still running.
 func Do(threshold time.Duration, sf SlowFunc, cb CallbackFunc) error {
```

Comment:
It's not really relevant to what this PR was about, but I was amused to notice this here because I recently wrote a very similar function elsewhere because I didn't realize we already had this one:

https://github.com/opentofu/opentofu/blob/af38729ab8750cea5465839ce2b2a1f226f7fdf4/internal/providercache/dir_modify.go#L188-L200

The two aren't quite interchangeable so I don't mean to say we ought to rush to consolidate them -- neither is very much code, after all -- but interesting to see the same problem solved in a very similar way in two places. 🤷‍♂️